octavia-cli/octavia_cli/templates/connection.yaml.j2

```diff
@@ -0,0 +1,42 @@
+name: {{ resource_name }}
+source_id: {{ source_id }}
+destination_id: {{ destination_id }}
+namespace_definition: source
+namespace_format: ${SOURCE_NAMESPACE}
+prefix: ""
+operation_ids:
+- 6c9e2330-4af3-47a9-b252-12fcfefcab55
+resource_requirements:
+  cpu_limit: ""
+  cpu_request: ""
+  memory_limit: ""
+  memory_request: ""
```

Comment:
Can you add comments to tell if these are required fields and share examples of how they should be filled?